src/pages/_document.js

```diff
@@ -40,6 +40,9 @@ export default class IntlDocument extends Document {
       GOOGLE_MAPS_API_KEY:
         process.env.GOOGLE_MAPS_API_KEY ||
         'AIzaSyCRLIexl7EkMQk_0_yNsjO4Vqb_MccD-RI',
+      RECAPTCHA_SITE_KEY:
+        process.env.RECAPTCHA_SITE_KEY ||
+        '6LcyeXoUAAAAAFtdHDZfsxncFUkD9NqydqbIFcCK',
```

Comment:
What is this default key?

Reply:
It's mostly invisible.  

![screen shot 2018-11-13 at 17 08 12](https://user-images.githubusercontent.com/806/48426293-c4d87280-e766-11e8-9199-1aebe98d50cd.png)
Just, once it's loaded there is a logo similar to this one appearing on the bottom of the page.

Reply:
It's a key I generated for us to use, at least on dev/staging. We may generate another one for production. Similar to what we're doing with Stripe.